env-check.sh

```diff
@@ -1,17 +1,20 @@
 #!/bin/bash
+set -e
 
 mydir=$(cd "$(dirname "$0")"; pwd)
 
 # enable dynamic debug logs for SOF modules
-DYNDBG="/etc/modprobe.d/sof-dyndbg.conf"
+dyn_dbg_conf="/etc/modprobe.d/sof-dyndbg.conf"
+sof_logger="/usr/bin/sof-logger"
+sof_ctl="/usr/bin/sof-ctl"
+sof_ldc="/etc/sof/sof-$("$mydir"/tools/sof-dump-status.py -p).ldc"
```

Comment:
we'd better provide a method to have some ENV override for ldc path. This is all for CI environment, we should make it more generic to normal user.